harness/determined/cli/rbac.py

```diff
@@ -89,9 +89,13 @@ def my_permissions(args: Namespace) -> None:
             workspace_name = bindings.get_GetWorkspace(sess, id=wid).workspace.name
             print(f"permissions assigned over workspace '{workspace_name}' with ID '{wid}'")
 
-        render.render_objects(
-            v1PermissionHeaders, [render.unmarshal(v1PermissionHeaders, p.to_json()) for p in perms]
-        )
+        perms_to_render = []
+        perms_added = set()
+        for p in perms:
+            if p.id not in perms_added:
+                perms_added.add(p.id)
+                perms_to_render.append(render.unmarshal(v1PermissionHeaders, p.to_json()))
+        render.render_objects(v1PermissionHeaders, perms_to_render)
```

Comment:
nit: instead of building a list all over again, you can just use `list(perms_added)`

Reply:
`perms_added` holds the permission ids while `perms_to_render` holds the full permissions converted to json (and ready for render). 

I tried just storing a set of json-converted permissions to have one less iterable
```
 perms_json: set[Dict [str, Any]] = set()
        for p in perms:
            p_json = p.to_json()
            if p_json not in perms_json:
                perms_json.add(p_json)
        render.render_objects(v1PermissionHeaders, 
                              [render.unmarshal(v1PermissionHeaders, p) for p in perms_json])
        print()
```
but `dicts` are unhashable so this doesn't work